motoman_driver/MotoPlus/MotionServer.c

```diff
@@ -1751,6 +1755,13 @@ void Ros_MotionServer_IncMoveLoopStart(Controller* controller) //<-- IP_CLK prio
 			{
 				if(ret == -3)
 					printf("mpExRcsIncrementMove returned: %d (ctrl_grp = %d)\r\n", ret, moveData.ctrl_grp);
+#if (YRC1000||YRC1000u)
+				else if (ret == -1 || ret == -19)

```

Comment:
I realise there is already one on line 1756, but could we avoid adding more magic numbers? If possible, please add named constants for `-1` and `-19`).

Reply:
I'll clarify.

Reply:
`-1` appears to be `E_EXRCS_IMOV_UNREADY`.

The YRC1000 API Spec I have does not list `-19`?

Reply:
I've removed the -19 of the condition.  It is not a valid return value.  I'm must have accidently mixted it up with some other function return value.